styles/globals.css

```diff
@@ -461,3 +461,15 @@ html {
         background-position: 0% 50%;
     }
 }
+
+kbd {
```

Comment:
@asaadam what is it for?

Reply:
<img width="69" alt="image" src="https://user-images.githubusercontent.com/1397612/178687904-f245042a-5dd0-4ea0-8a9c-83ee8d0a02b2.png">
for this one ser